zivid_camera/test/test_zivid_camera.test

```diff
@@ -2,5 +2,6 @@
     <node name="zivid_camera" pkg="zivid_camera" type="zivid_camera_node" ns="zivid_camera" output="screen">
         <param name="file_camera_path" type="str" value="/usr/share/Zivid/data/FileCameraZividOne.zfc" />
     </node>
-    <test test-name="zivid_camera_test" pkg="zivid_camera" type="zivid_camera_test" time-limit="300.0" />
+    <test test-name="zivid_camera_test" pkg="zivid_camera" type="zivid_camera_test" time-limit="300.0"
+        launch-prefix="bash -c 'rosservice call --wait /zivid_camera/zivid_camera/set_logger_level ros.zivid_camera debug; $0 $@' " />
```

Comment:
Was hoping there was a nice way to set the verbosity level directly on the node in the launch file, but it does look like it exists.

What do you think about doing this instead of the launch-prefix?:
```
<node pkg="rosservice" type="rosservice" name="set_zivid_camera_logger_level" args="call --wait /zivid_camera/zivid_camera/set_logger_level 'ros.zivid_camera' 'debug'" />
```

Reply:
i tried something like that first and could not get it to work. Have you tried this on your PC?

Reply:
Actually it does work, the only problem is that since nodes are launched in parallell, we could be starting tests before the logger level is set. Which is the main reason that I did it like this, first enabling logging, then running the tests.

Reply:
Ah, I see. Let's keep it as is then, so we're sure it's the right order :+1: